apps/notifications/src/panel/rest-client/index.js

```diff
@@ -211,17 +205,17 @@ function getNotes() {
 
 		store.dispatch( actions.ui.loadedNotes() );
 
-		const oldNotes = getAllNotes( store.getState() ).map( property( 'id' ) );
-		const newNotes = data.notes.map( property( 'id' ) );
-		const notesToRemove = difference( oldNotes, newNotes );
+		const oldNotes = getAllNotes( store.getState() ).map( ( n ) => n.id );
```

Comment:
Nit: Or if we wanna avoid declaring `n` we could destructure `id`, although it ends up a couple of characters longer:

```suggestion
		const oldNotes = getAllNotes( store.getState() ).map( ( { id } ) => id );
```